codex/deploy.yml

```diff
@@ -0,0 +1,103 @@
+---
+version: "2.0"
+
+endpoints: 
+  nodeip:
+    kind: ip
+
+services:
+  codex:
+    image: codexstorage/nim-codex:0.1.6
+    expose:
+#    - port: 8080 # You probably don't want to expose the API globally:) Consider using reverse proxy with some auth?
+#      as: 8080
+#      to:
+#        - global: true
+#          ip: nodeip
+    - port: 8070
+      as: 8070
+      proto: tcp
+      to:
+        - global: true
+          ip: nodeip
+    - port: 8090
+      as: 8090
+      proto: udp
+      to:
+        - global: true
+          ip: nodeip
+    env:
+      - DATA_DIR=/codex/data
+      - PRIV_KEY=
+      - CODEX_NAT=
+      - CODEX_LOG_LEVEL=debug
+      - CODEX_BLOCK_TTL=30d 
+    command:
+    - sh
+    - -c
+    args:
+    - >
+        if [ -z "${CODEX_NAT}" ]; then
+          echo "Missing \$CODEX_NAT=public_ip_of_the_deplyoment"
+          sleep 120
+          exit 1
+        fi
+
+        mkdir -p ${DATA_DIR}
+        chmod 0700 ${DATA_DIR}
+
+        quota=$(( $(df --output=size /codex | tail -1) * 1000 ))
```

Comment:
```suggestion
        quota=$(df --output=size --block-size=1 /codex | tail -1)
```
I was going to suggest 1024 instead of 1000 but then realized a blocksize of 1 means you don't even have to think about the conversion.